packages/rule-manifest-exists/README.md

```diff
@@ -1,17 +1,18 @@
 # Require a web app manifest file (`@sonarwhal/rule-manifest-exists`)
 
-`manifest-exists` warns against not providing a [web app manifest][spec]
-file.
+`manifest-exists` warns when a [web app manifest][spec]
+file is not found.
 
 ## Why is this important?
 
-The web app manifest file constitutes a standard centralized place
-to put metadata about your site/web app, and providing it:
+The web app manifest file is required to define a progressive web
+app. It's the standard, centralized store of the applications
+metadata which:
```

Comment:
I'm going to keep most of the initial text as it was taken from the [spec](https://www.w3.org/TR/appmanifest/#abstract-0)

> centralized place to put metadata associated with a web application